internal/exec/describe_affected.go

```diff
@@ -125,16 +146,78 @@ func ExecuteDescribeAffectedCmd(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	if verbose {
-		cliConfig.Logs.Level = u.LogLevelTrace
-	}
-
 	u.LogTrace(cliConfig, fmt.Sprintf("\nAffected components and stacks: \n"))
 
 	err = printOrWriteToFile(format, file, affected)
 	if err != nil {
 		return err
 	}
 
+	// Upload the affected components and stacks to a specified endpoint
+	// https://www.digitalocean.com/community/tutorials/how-to-make-http-requests-in-go
+	if upload && len(affected) > 0 {
```

Comment:
We should upload an empty `stacks: []` even if len==0, otherwise atmos pro will be stuck waiting for the affected graph to be uploaded and won't be able to know that there were no changes.